dev/bots/test.dart

```diff
@@ -248,7 +249,7 @@ Future<void> main(List<String> args) async {
       // All web integration tests
       'web_long_running_tests': () => webLongRunningTestsRunner(flutterRoot),
       'flutter_plugins': _runFlutterPackagesTests,
-      'skp_generator': _runSkpGeneratorTests,
+      'skp_generator': () => skpGeneratorTestsRunner(),
```

Comment:
nit, you don't need the anonymous closure here, you could just tear off:

`      'skp_generator': skpGeneratorTestsRunner,`

Reply:
Updated! Thank you